### ISSUE TITLE
Does index.html requires main.js

### ISSUE THREAD

**Issue by hk-skit** [Peleke/browserify_introduction#2]
Does index.html needs main.js. I mean all the dependencies have been bundled by browserify. So only bundle.js js should be included. Please correct me if I am wrong.

**Comment by rgdonohue** [comment]
I agree. The client will throw an error if main.js is still included in the build?
